pywr/solvers/cython_glpk.pyx

```diff
@@ -343,7 +357,10 @@ cdef class CythonGLPKSolver(GLPKSolver):
             elif isinstance(some_node, Storage):
                 storages.append(some_node)
             elif isinstance(some_node, AggregatedNode):
-                if some_node.factors is not None:
+                if some_node.factor_parameters is not None:
+                    aggregated_with_factor_params.append(some_node)
+                    some_node.__agg_factor_data = AggNodeFactorData()
```

Comment:
Please add `__agg_factor_data` to the definition of the `AggregatedNode` in a similar way to the `__data` attribute.